pyproject.toml

```diff
@@ -6,10 +6,24 @@ requires = [
 ]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "equistore"
+description = "A specialized data storage format suited to atomistic machine learning needs and more."
+readme = "README.rst"
+requires-python = ">=3.6"
+
+dependencies = [
+    "numpy>=1.20.0",
+]
+dynamic = ["version"]
```

Comment:
what does this do?

Reply:
This tells the build system the our version is dynamically set in the `setup.py`.